app/containers/RoomHeader/index.tsx

```diff
@@ -139,29 +94,28 @@ class RoomHeaderContainer extends Component<IRoomHeaderContainerProps, any> {
 				sourceType={sourceType}
 			/>
 		);
-	}
-}
-
-const mapStateToProps = (state: IApplicationState, ownProps: any) => {
-	let statusText = '';
-	let status = 'offline';
-	const { roomUserId, type, visitor = {}, tmid } = ownProps;
-
-	if (state.meteor.connected) {
-		if ((type === 'd' || (tmid && roomUserId)) && state.activeUsers[roomUserId]) {
-			({ status, statusText } = state.activeUsers[roomUserId]);
-		} else if (type === 'l' && visitor?.status) {
-			({ status } = visitor);
+	},
+	(prevProps, nextProps) => {
```

Comment:
Do we still need this?

Reply:
Yes and no. Let me explain,

For a normal user/channel/room the RoomHeader will be rendered only once when open the room. But, for omnichannel's chat the roomheader re-render 3 times when open the room! WDYT? Keep the check difference or just remove them?